pymove/utils/trajectories.py

```diff
@@ -324,11 +323,11 @@ def column_to_array(df_, label_conversion):
                 'Dataframe must contain a %s column' % label_conversion
             )
 
-        arr = np.full(df_.shape[0], None, dtype=np.ndarray)
-        for idx, row in progress_bar(df_.iterrows(), total=df_.shape[0]):
-            arr[idx] = object_for_array(row[label_conversion])
-
-        df_[label_conversion] = arr
+        df_[label_conversion] = (
+            df_[label_conversion].apply(
+                lambda x: object_for_array(x)
```

Comment:
Codacy found an issue: [Lambda may not be necessary](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6419254)